diff --git a/node_modules/expo-device/ios/UIDevice.swift b/node_modules/expo-device/ios/UIDevice.swift
index 7d58ee8..f157ba1 100644
--- a/node_modules/expo-device/ios/UIDevice.swift
+++ b/node_modules/expo-device/ios/UIDevice.swift
@@ -1,3 +1,5 @@
+import Foundation
+import TargetConditionals
 import UIKit
 
 struct ExpoDeviceType {
@@ -185,7 +187,11 @@ public extension UIDevice {
 
   // Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
   var isSimulator: Bool {
-    return TARGET_OS_SIMULATOR != 0
+    #if targetEnvironment(simulator)
+      return true
+    #else
+      return false
+    #endif
   }
 
   var isJailbroken: Bool {
